attic/lambdas/service/app.py

```diff
@@ -0,0 +1,704 @@
+@app.authorizer()
```

Comment:
I feel this file should preserve the import statements referenced in the moved code, otherwise we're not really preserving it.

Reply:
Hard to tell whether this is correct advice or not without going every file in the attic. It just seems to me that there's a huge number of classes referenced in this code here that might be ambiguous to resolve in the future.

Reply:
The main reason for the attic is not so much archival but maintaining a list of what used to exist. If you we just deleted files, it would be harder to find the commit that deleted a particular without knowing where it used to exist and what its name was. 

To get to the *exact* original, one can always look at the commit that placed the file into the attic.